olad/OladHTTPServer.cpp

```diff
@@ -219,6 +219,155 @@ OladHTTPServer::OladHTTPServer(ExportMap *export_map,
   m_server.RegisterFile(
       "/new/img/logo-mini.png",
       HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "olad/www/new/img/logo-mini@2x.png",
```

Comment:
the path of the file is incorrect it should be ```/new/img/logo-mini@2x.png``` instead of ```olad/www/new/img/logo-mini@2x.png```. This is also true for the rest of the files you added